CHANGELOG.md

```diff
@@ -13,6 +14,12 @@
 | 1.1.1        |   9/16/21  | Updated the extension to support ARIA 1.2 and ARIA in HTML requirements
 | 0.96.0       |  10/30/19  | Fixes bug in highlight option not being remembered
 
+
+### 2.2.0
+* Added information button to rule summary and element summary grids to provide documentation on the meaning of the symbols
+* Updating labeling of views to improve indemnification of information in each view
```

Comment:
Not sure why the word 'indemnification' was used here. Would suggest: 'Updated labeling of views to provide more context for the information in each view'